### DISCUSSION TITLE
GTP-U packet dropped

### DISCUSSION THREAD

**Discussion by ThorisoMakhosa** [srsran/srsRAN_Project#197] [General Help]
Hi Experts

May you please assist in the analysis of this log of which i pasted the screenshot below. I keep on getting the Dropped GTP-U packet error after the UE transmits on the PUCCH. What could be reason? is it attributed to channel conditions?

![image](https://github.com/srsran/srsRAN_Project/assets/107929145/e9a59726-a48d-46a4-8e87-46900d434dc0)

**Comment by robertfalkenberg** [comment]
These GTPU errors are unrelated to the PUCCH. These are DL packets coming from the core for an expired PDU session.
The fact that there is `rnti=0x4604`  and the gNB starts with `0x4601` is also a strong indication for this.

For further assistance, please share NGAP pcap and the full log as textfile, together with some general information about your setup.
